examples/spring_boot/maven_install.json

```diff
@@ -1,1483 +1,1459 @@
 {
-    "dependency_tree": {
```

Comment:
You appear to have repinned this file using an older version of `rules_jvm_external`. Could you please rebase on `HEAD` of `master` and repin again?